src/sitemapper.cr

```diff
@@ -19,36 +19,47 @@ module Sitemapper
     setting host : String, example: "https://mysite.com"
     setting sitemap_host : String? = nil
     setting max_urls : Int32 = 500
-    setting storage : Symbol = :local
+    setting storage : Sitemapper::Storage.class = Sitemapper::LocalStorage
```

Comment:
Instead of using a Symbol then a case statement, just point to the class, and we can instantiate it directly.